PF1/Assignment3/LongestSequence.java

```diff
@@ -0,0 +1,64 @@
+package assignment3;
+
+// Find longest increasing subsequence from an array of integers
+public class LongestSequence {
+	
+	/* store length of last longest occuring subsequence
+	 * compare with evry ending sub-sequence
+	 * */
+	public int[] longestSequence(int[] input){
+		if(input.length <=1) 
+			return input;
+		
+		int[] tmp= new int[input.length];
+		int k=0;
+		
+		int max=0;
+		int start_marker = 0, end_marker = 0;
+		
+		int i=0;
+		for(i= start_marker; i < input.length-1; i++){
+			if(input[i] > input[i+1]){
+				System.out.println("n: "+ i+ " "+ input[i] + " "+ input[i+1]);
```

Comment:
Operational method should not have any method to print on console. method should only have calculations and other business logic. if it is necessary then you can use.